nltk/tag/mallet.py

```diff
@@ -0,0 +1,249 @@
+# -*- coding: utf-8 -*-
+# Natural Language Toolkit: Interface to the Stanford NER-tagger
+#
+# Copyright (C) 2001-2015 NLTK Project
+# Author: Long Duong <longdt219@gmail.com>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+"""
+A module for interfacing with the Mallet CRF taggers.
+"""
+
+import os
+import tempfile
+from subprocess import PIPE
+import unicodedata
+import re 
+
+from nltk.internals import find_jar, config_java, java, _java_options
+from nltk.tag.api import TaggerI
+
+_mallet_url = 'http://mallet.cs.umass.edu/grmm/'
+
+class MalletTagger(TaggerI):
+    """
+    An interface to Mallet CRF taggers.
+    
+    >>> from nltk.tag.mallet import MalletTagger
+    >>> mt = MalletTagger('/Users/HeroAthen/Dropbox/Work/nltk/temp/mallet-2.0.7/dist/mallet.jar', 
```

Comment:
The test shouldn't depend on hardcoded Mallet paths - it will fail on CI server and on users' machines.